utils/build_swift/driver_arguments.py

```diff
@@ -972,6 +972,10 @@ def create_argument_parser():
         help="the absolute path to libtool. Default is auto detected.",
         type=arguments.type.executable,
         metavar="PATH")
+    parser.add_argument(
```

Comment:
I think build-script options are supposed to match build-script-impl whenever possible, so this should be `--use-lld-linker`. (That also makes sense because it's conceivable that the option would *build* lld, since it's an LLVM project.)

Reply:
that makes sense. I'll change that over. 

I actually added another arg locally that I'm going to push soon as part of fixing things so that we can cross compile from a Darwin toolchain to Linux/Fuchsia/BSD to be able to use lld only for the stdlib for specific sdk targets. It's a bit funky how that maps though though the python/shell/cmake layers since we don't have any sdk specific targeting args from the buid-util anywhere else. Looks something like: 

```--use-lld-for-stdlib=linux-x86_64,linux-aarch64,fuchsia-x86_64,fuchsia-aarch64```

Reply:
fixed. now is --use-lld-linker in frontend script.